### PR TITLE
bugfix/FOUR-11082: AI Assitant menu now resets after applying changes

### PR DESCRIPTION
**Pull request by TinCodes**
## Issue & Reproduction Steps
Upon applying changes in Script Editor using AI Generated code, the AI Assistant Menu should revert to its original state (the AI Menu showing the 4 AI Tools). 

Expected behavior:
The menu should return and display the options for the script

Current behavior:
After applying the changes the menu does not return to the main menu, it stays on the description to generate

## Solution
Since there is no variable or attribute that could trigger the AI menu to go back from `ScriptEditor.vue`, a `ref` attribute was added to the AI Tab components. These refs help to gain access to the `AiTab.vue` component's method `showMenu()`, which works exactly as the arrow icon to go back to the main AI menu,.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11082

ci:next
ci:deploy

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3b2c0ca40f.eng.processmaker.net

File: resources/js/processes/scripts/components/ScriptEditor.vue [hunk 1]
```diff
@@ -106,6 +106,7 @@
               <b-card-body class="overflow-hidden p-0">
                 <b-list-group class="w-100 h-100 overflow-auto">
                   <cornea-tab
```
Comment (agustinbusso) [inline resources/js/processes/scripts/components/ScriptEditor.vue:108]: Can you replace this with `<ai-tab` in a similar way to this PR please?
https://github.com/ProcessMaker/processmaker/pull/5477/files

**Review by agustinbusso — CHANGES_REQUESTED** [review]

**Review by agustinbusso — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3b2c0ca40f.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3b2c0ca40f.eng.processmaker.net

**Pull request merged**